tests/flow_type_spread/jsfmt.spec.js

```diff
@@ -0,0 +1,2 @@
+run_spec(__dirname);
+run_spec(__dirname, {parser: 'flow'});
```

Comment:
Woops, good call! I'll fix this :) I tested it on the command line and it's working on both.